modes/help/evil-collection-help.el

```diff
@@ -45,15 +45,16 @@
     (kbd "C-b") 'scroll-down-command
     (kbd "<tab>") 'forward-button
     (kbd "<backtab>") 'backward-button
+    ;; This exists because tab key isn't recognized as <tab> on terminal.
```

Comment:
I think we should pause on this until we come to a consensus in https://github.com/emacs-evil/evil-collection/issues/290.

Reply:
The concensus cannot be reached if somebody doesn't propose something. This pull request is a proposal.